front/src/components/map/KakaoMap.vue

```diff
@@ -61,6 +55,46 @@ export default {
       const address = Object.values(centerAddress).join(" ");
       this.$store.commit("appBar/CHANGE_ADDRESS", address);
     },
+    async checkMemberLocationInformation() {
+      await this.$kakaoMap
+        .getCurrentLocation()
+        .then(() => {
+          this.$store.commit("member/SET_LOCATION_INFORMATION_TRUE");
+        })
+        .catch(() => {});
+    },
+    async checkMemberEnvironment() {
+      let isMobile = false;
+      const filter = "win16|win32|win64|mac|macIntel";
+      if (navigator.platform) {
+        isMobile = filter.indexOf(navigator.platform.toLowerCase()) < 0;
```

Comment:
```suggestion
        isMobile = filter.includes(navigator.platform.toLowerCase());
```

이건 안되나요?
근데 `macIntel`은 대문자가 들어가는데 `LowerCase`로 바꿔도 괜찮나요?

Reply:
같은 일을 하지만 코드 의도를 더 명확하게 해주네요! 반영하겠습니다.

Reply:
같은 기능을 하지만 코드 의도가 더 명확해지네요! 반영하겠습니다.